pkg/plugins/policies/core/matchers/testdata/egressmatchedpolicies/meshexternalservice/torules/01.mes.yaml

```diff
@@ -0,0 +1,11 @@
+type: MeshExternalService
```

Comment:
If these are the first tests of a kind then it's easier to have MES, MeshHealthChecks and Meshes in a single input file. See the util code here https://github.com/kumahq/kuma/blob/fca161a8bb09b00662a7b67a416060c019ac1975/pkg/plugins/policies/core/rules/suite_test.go#L22